packages/image_picker/ios/Classes/ImagePickerPlugin.m

```diff
@@ -136,4 +141,17 @@ - (UIImage *)normalizedImage:(UIImage *)image {
   return normalizedImage;
 }
 
+- (void)storeFile:(NSString *)filePath {
+  NSData *data = [[NSData alloc] initWithContentsOfFile:filePath];
+  UIImage *image = [UIImage imageWithData:data];
+  UIImageWriteToSavedPhotosAlbum(image, nil, @selector(storeComplete), nil);
```

Comment:
I'm not sure how this is supposed to work... Do you mean to say `UIImageWriteToSavedPhotosAlbum(image, self, @selector(storeComplete:didFinishSavingWithError:contextInfo:), nil)` here?